src/start

```diff
@@ -26,6 +26,10 @@ fi
 popd
 cz-install bootmenu `cat /mnt/state/volume-number`
 
+# C1 V-260570 Ubuntu 22.04 LTS must not allow accounts configured with blank or null passwords.
+# https://stigviewer.com/stig/canonical_ubuntu_22.04_lts/2024-11-25/finding/V-260570
+sed -i 's/nullok//g' /etc/pam.d/common-auth
```

Comment:
Have you tried this? We do like that on cloud environments because there is no password set and access happens via pubkey.

This could break on those environments.